packages/compile-solidity/src/run.ts

```diff
@@ -140,19 +205,13 @@ function orderABI({ abi, contractName, ast }) {
   ];
 }
 
-/**
- * @param sources - { [sourcePath]: contents }
- * @param targets - sourcePath[]
- * @param setings - subset of Solidity settings
- * @return solc compiler input JSON
- */
```

Comment:
Why remove the typedoc?

Reply:
Ah, I guess they weren't all that useful before. Might be nice if you were to update them, though.

That said, consider this an optional request.

Reply:
It seemed redundant now that we have the types...plus TS told me to get rid of it :)

Reply:
Yeah if there isn't more to say other than type details, I suppose it would be.